installer/sharch_body.sh

```diff
@@ -25,13 +25,19 @@ fi
 
 echo " OK."
 
+image_size=$(du "$0" | awk '{print $1}')
 # Untar and launch install script in a tmpfs
 cur_wd=$(pwd)
 export cur_wd
 archive_path=$(realpath "$0")
 tmp_dir=$(mktemp -d)
 if [ "$(id -u)" = "0" ] ; then
     mount -t tmpfs tmpfs-installer $tmp_dir || exit 1
+    mount_size=$(df $tmp_dir | tail -1 | tr -s ' ' | cut -d' ' -f4)
+    if [ "$mount_size" -lt "$((image_size*3))" ]; then
```

Comment:
>image_size*3 [](http://example.com/codeflow?start=31&length=12)

This seems an estimation. How do you know *3 is enough? Could you get the accurate required size?

Reply:
that must be generated from the build process.

Reply:
use this to get the original size?

lgh@p330:~$ gzip -l docker-sonic-vs.gz 
         compressed        uncompressed  ratio uncompressed_name
          242618472           684168704  64.5% docker-sonic-vs

Reply:
This script untars sonic-<platform>.bin, which is a ONIE image. We cannot use gzip command. 
Will update the logic by getting image size from the "tar" command, excluding the ONIE installer script